algorithms/CSharp/test/Search/linear-search.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using NUnit.Framework;
+using Algorithms.Search;
+
+namespace Algorithms.Tests.Search
+{
+    [TestFixture]
+    class LinearSearchTest
+    {
+        int[] numbers = { 1, 2, 3, 4, 5 };
+
+        [Test]
+        [TestCase(4, 3)]
+        [TestCase(5, 4)]
+        public void Should_Return_Index_Of_Matching_Element_In_Array(int searchNumber, int expectedIndexOfSearchTerm)
```

Comment:
The method name is `InPutCodition_OutPutCodition` example `PassSearchNumber_ShouldGetExpectedIndex`